src/HW_models/irq_sources.h

```diff
@@ -10,6 +10,33 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#if defined(CONFIG_SOC_SERIES_NRF53X) || \
```

Comment:
I believe it is sufficient to only keep ``#if defined(CONFIG_SOC_COMPATIBLE_NRF53X_CPUNET)`